nevergrad/benchmark/experiments.py

```diff
@@ -892,6 +893,20 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                     if not xp.is_incoherent:
                         yield xp
 
+@registry.register
+def adversarial_images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Adversarial attacks."""
+    seedg = create_seed_generator(seed)
```

Comment:
So we create resnet50(pretrained=True)   here ?

Reply:
No, it should be in https://github.com/facebookresearch/nevergrad/pull/646/files#diff-9237fdd46c25be2ca72b153f11ed0cebR130 ;)